Update adding_test_data.md (#3764)
chef/automate: dev-docs/adding-data/adding_test_data.md

from hab studio:

> there are two opions...

Reviewer: ```suggestion
there are two options...
```

1. `chef_load_nodes` 

1. to load infra nodes with failed runs:

>     ```

Reviewer: ```suggestion
    ```shell
```

from hab studio:

there are three options...

1. `chef_load_compliance_scans -N100 -D4 -M3 -T1000`

    100 nodes, scanning for the past 4 days, max of 3 node scans per day, no more than 1000 total scans. modify the numbers on each of those options to meet your needs.
1. `chef_load_compliance_nodes`

    runs the above command with the following easy default options: `-N50 -D10 -M1 -T500`
1. `load_compliance_reports`

>     will send the reports located in `components/compliance-service/test-data/audit-reports` to your dev enironment. these reports are populated with fields such as chef_server, organization, policy_name. these reports are especially useful when you need to test searchbar functionality for multiple search fields or resource scoped access ingest match rules.

Reviewer: ```suggestion
    will send the reports located in `components/compliance-service/test-data/audit-reports` to your dev environment. these reports are populated with fields such as chef_server, organization, policy_name. these reports are especially useful when you need to test search bar functionality for multiple search fields or resource scoped access ingest match rules.
```

## Adding cloud integrations

AWS:

>  - use the `okta_aws` functionality to get your temporary aws credentials [see details here](https://chefio.atlassian.net/wiki/spaces/ENG/pages/94259373/SOP-001+-+Gaining+access+to+okta+AWS+accounts+via+the+AWS+API#SOP-001-GainingaccesstooktaAWSaccountsviatheAWSAPI-Step1:Installokta_aws)

Reviewer 2: minor link tag updates

>  - use the `okta_aws` functionality to get your temporary aws credentials [see details here](https://chefio.atlassian.net/wiki/spaces/ENG/pages/94259373/SOP-001+-+Gaining+access+to+okta+AWS+accounts+via+the+AWS+API#SOP-001-GainingaccesstooktaAWSaccountsviatheAWSAPI-Step1:Installokta_aws)

Author: ```suggestion
 - use the `okta_aws` functionality to get your temporary [aws credentials](https://chefio.atlassian.net/wiki/spaces/ENG/pages/94259373/SOP-001+-+Gaining+access+to+okta+AWS+accounts+via+the+AWS+API#SOP-001-GainingaccesstooktaAWSaccountsviatheAWSAPI-Step1:Installokta_aws)
```

ensure you have `jq` installed

there are three options...

> 1. `infra_service_load_sample_data`

Reviewer: I would also add a warning on (1) and (2) explicitly: "these will result in 'invalid org ID' messages if you drill down to organizations; see (3) to avoid that" -- if I am interpreting what you said for (3) correctly.

    
    By default, 50 records of the infra servers and orgs will be added with the prefix `chef-server` and `chef-org` respectively.

> 1. `infra_service_load_sample_data -N 100 -S infra-server -O infra-org`

Reviewer: I like what you did above, so to be more parallel, reverse these two and describe the (now) second one with "runs the above command with the following easy default options:..."

    
    100 records of the infra servers and orgs will be added, modify the numbers on each of those options to meet your needs.

> 1. [internal steps](https://chefio.slack.com/archives/CPYNX33LJ/p1583485764104300) this is the only method that works against a running chef server and will allow you to get data that does not result in the cookbook error

Reviewer: Is "the cookbook error" what I referred to? If so, it is actually an org error; it will happen with any of cookbook's siblings, too (environment, data bags, etc.)--all because they are children of the org. Might mention the error phrase, too: "invalid org ID: private key block size invalid"

Reviewer 2: @msorens `invalid org ID: private key block size invalid` would be the better user-readable message in future As I am correctly working on.

will commit changes to mentioned `(environment, data bags, etc.)` as well

thanks
